closure/testing/test/BUILD

```diff
@@ -31,28 +31,31 @@ closure_js_library(
     deps = ["//closure/library"],
 )
 
-closure_js_test(
-    name = "arithmetic_module_test",
-    srcs = ["arithmetic_module_test.js"],
-    defs = ["--entry_point=goog:arithmetic_module_test"],
-    language = "ECMASCRIPT6_STRICT",
-    deps = [
-        ":arithmetic_module_lib",
-        "//closure/library",
-        "//closure/library:testing",
-    ],
-)
-
-closure_js_test(
-    name = "arithmetic_scope_test",
-    srcs = ["arithmetic_scope_test.js"],
-    language = "ECMASCRIPT6_STRICT",
-    deps = [
-        ":arithmetic_module_lib",
-        "//closure/library",
-        "//closure/library:testing",
-    ],
-)
+# TODO(hochhaus): Enable test when upstream local file loader fix is included
```

Comment:
I think that setting goog.ENABLE_DEBUG_LOADER = false in phantomjs_runner.js along with a strict goog.module/goog.require behavior was the real cause of the test failure.

https://github.com/google/closure-library/wiki/goog.module:-an-ES6-module-like-alternative-to-goog.provide#how-do-i-use-a-googmodule-from-a-traditional-closure-file

I think I have addressed all issues with this PR now but I will check again in the morning when I have a fresh set of eyes.

Please hold off on review for now...